apps/minifront/src/components/swap/swap-form/token-swap-input.tsx

```diff
@@ -63,11 +67,11 @@ const tokenSwapInputSelector = (state: AllSlices) => ({
 export const TokenSwapInput = () => {
   const status = useStatus();
   const latestKnownBlockHeight = status.data?.latestKnownBlockHeight ?? 0n;
-  const balancesResponses = useBalancesResponses();
```

Comment:
This version of `useBalancesResponses()` was only fetching swappable balances responses. The new `useBalancesResponses()` fetches _all_ balances responses, so we now use the `swappableBalancesResponsesSelector` to filter for just the balances we want.